### ISSUE TITLE
Installation Difficulties

### ISSUE THREAD

**Issue by ramav87** [gpufit/Gpufit#103]
Hi, 

I am trying to install this on my Linux server but have run into an error. I ran the command 

`cmake -DCMAKE_BUILD_TYPE=RELEASE ../Gpufit
`
which worked, and then I tried

`make`

but it seems to give this error:

```
[ 18%] Built target Cpufit
[ 53%] Built target Gpufit
[ 56%] Building CXX object Gpufit/examples/CMakeFiles/Linear_Regression_Example.dir/Linear_Regression_Example.cpp.o
/home/rvv/RLwork/AEwork/Gpufit/examples/c++/Linear_Regression_Example.cpp:1:10: fatal error: ../../gpufit/gpufit.h: No such file or directory
 #include "../../gpufit/gpufit.h"
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Gpufit/examples/CMakeFiles/Linear_Regression_Example.dir/build.make:65: recipe for target 'Gpufit/examples/CMakeFiles/Linear_Regression_Example.dir/Linear_Regression_Example.cpp.o' failed
make[2]: *** [Gpufit/examples/CMakeFiles/Linear_Regression_Example.dir/Linear_Regression_Example.cpp.o] Error 1
CMakeFiles/Makefile2:1116: recipe for target 'Gpufit/examples/CMakeFiles/Linear_Regression_Example.dir/all' failed
make[1]: *** [Gpufit/examples/CMakeFiles/Linear_Regression_Example.dir/all] Error 2
Makefile:97: recipe for target 'all' failed
make: *** [all] Error 2

```

I'm not sure how to proceed here.

**Comment by ewantarrant** [comment]
I have the same problem with current master (af1389c6c7a8a2e7eb7b0ccb1d7595d18880e57c). The issue is that some relative `#include` paths are not correct.

```
# Clone
$ git clone https://github.com/gpufit/Gpufit.git Gpufit

# Offending header files
$ grep -nr "gpufit/gpufit.h" --include *.h --include *.cpp --include *.cu
Gpufit/examples/c++/Gauss_Fit_2D_Example.cpp:1:#include "../../gpufit/gpufit.h"
Gpufit/examples/c++/CUDA_Interface_Example.cu:1:#include "../../gpufit/gpufit.h"
Gpufit/examples/c++/Linear_Regression_Example.cpp:1:#include "../../gpufit/gpufit.h"
Gpufit/examples/c++/Simple_Example.cpp:1:#include "../../gpufit/gpufit.h"
```

These includes should read `#include "../../Gpufit/gpufit.h"` (note upper case `G`).

If you correct this mistake and re-compile it might work.

The alternative is to 

```
git clone https://github.com/gpufit/Gpufit.git Gpufit && git checkout v1.2.0
```

which does not suffer this problem.

**Comment by superchromix** [comment]
Fixed in abce4a15032ccf3a7291c5c8798198903f84e5ec
